### PR TITLE
Ph5tostationxml sacpz

### PR DESCRIPTION
**Pull request by derick-hess**
Adds SACPZ and stationtxt to command line options

### CODE REVIEW

File: ph5/clients/ph5tostationxml.py [hunk 1]
```diff
@@ -910,6 +907,10 @@ def main():
                       nsmap={'iris': "http://www.fdsn.org/xml/station/1/iris"})
         elif out_format == "KML":
             inv.write(args.outfile, format='KML')
+        elif out_format == "SACPZ":
+            inv.write(args.outfile, format="SACPZ")
+        elif out_format == "STATIONTXT":
```
Comment (nick-iris) [inline ph5/clients/ph5tostationxml.py]: I would change this to "TEXT" in order to remain consistent with the web services.

Comment (derick-hess) [reply ph5/clients/ph5tostationxml.py]: good idea. I'll fix that.

**Comment by nick-iris** [comment]
I would check to see if any of the changes broke the flake8 test.

**Comment by derick-hess** [comment]
Made sure it passed flake8 already :)

**Comment by derick-hess** [comment]
Yeah I think for the 4.1.0 release we should probably rename it ph5toresponse. I'm going to also add the option to write out the RESP files as well. The data group here though that would be useful.

**Comment by nick-iris** [comment]
I don't think "ph5toresponse" would be good either since StationXML isn't exactly the instrument response, although it can include it. Something like "ph5tostationmetadata" would be more appropriate, but is kind of long.

**Comment by derick-hess** [comment]
Yeah, that's true. We have some time to think of a good name before the 4.1.0 release

**Pull request merged**